ci/make_geography_db.py

```diff
@@ -62,7 +62,10 @@ def make_geography_db(
             cols = ibis_schema.names
             path = Path(d, f"{table_name}.jsonl")
             path.write_text(
-                "\n".join(json.dumps(dict(zip(cols, row))) for row in data[table_name])
+                "\n".join(
+                    json.dumps(dict(zip(cols, row, strict=False)))
```

Comment:
Kind of annoying that this happened, but the reason is that https://docs.astral.sh/ruff/rules/zip-without-explicit-strict/ is available when targeting 3.10.

Guess these could mostly be `strict=True`, or could just ignore B905 for stylistic reasons.

Reply:
I'd vote to just disable that rule.

Reply:
and done!